dll/ntdll/ldr/ldrpe.c

```diff
@@ -976,13 +981,14 @@ LdrpSnapThunk(IN PVOID ExportBase,
     PANSI_STRING ForwardName;
     PVOID ForwarderHandle;
     ULONG ForwardOrdinal;
+    PROSCOMPAT_DESCRIPTOR RosCompatDescriptor;
 
```

Comment:
Adding `PVOID ExportBase = ExportLdrEntry->Base;` here will reduce the diff a bit.